PurchasesTests/Purchasing/EntitlementInfosTests.swift

```diff
@@ -472,23 +472,23 @@ class EntitlementInfosTests: XCTestCase {
         stubResponse(entitlements: mockEntitlements,
                      subscriptions: mockSubscriptions(ownershipType: "PURCHASED"))
 
-        var subscriberInfo = try XCTUnwrap(CustomerInfo(testData: response))
+        var subscriberInfo = try CustomerInfo(data: response)
         var entitlement = try XCTUnwrap(subscriberInfo.entitlements.active["pro_cat"])
 
         expect(entitlement.ownershipType) == .purchased
 
         stubResponse(entitlements: mockEntitlements,
                      subscriptions: mockSubscriptions(ownershipType: "FAMILY_SHARED"))
 
-        subscriberInfo = try XCTUnwrap(CustomerInfo(testData: response))
+        subscriberInfo = try CustomerInfo(data: response)
         entitlement = try XCTUnwrap(subscriberInfo.entitlements.active["pro_cat"])
 
         expect(entitlement.ownershipType) == .familyShared
 
         stubResponse(entitlements: mockEntitlements,
                      subscriptions: mockSubscriptions(ownershipType: "BOATY_MCBOATFACE"))
 
-        subscriberInfo = try XCTUnwrap(CustomerInfo(testData: response))
+        subscriberInfo = try CustomerInfo(data: response)
```

Comment:
These are no longer using the test-only `.init(testData:)`. The actual method in the code `.init(data:)` already has a `throws` initializer, so don't need to use `XCTUnwrap`. I think using this we can probably get rid of `init(testData:)`.